rules/es6.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
@shouston3 _thanks_ for adding this file. 
It would be good to create _issues_ as the ES6 features get requested/discussed. 👍 
(_so for now, this is perfect_)